song_rater/tests.py

```diff
@@ -10,9 +10,31 @@ def test_home_page(self):
         self.assertContains(response, 'Welcome to the homepage of Ordina Music Rater')
 
 
+def add_song(artist: str, title: str) -> Song:
+    """Helper function to add a song to the database"""
+    return Song.objects.create(artist=artist, title=title)
+
+
+def add_rating(song: Song, rating: int) -> None:
+    """Helper function to add a rating for a song to the database"""
+    song.ratings.create(rating=rating)
+
+
+def add_song_and_rating(artist: str, title: str, rating: int) -> Song:
+    """Helper function to add a song and a rating for the same song"""
+    song = add_song(artist=artist, title=title)
+    add_rating(song=song, rating=rating)
+
+    return song
+
+
 class AddSongTestCase(TestCase):
     def setUp(self):
-        Song.objects.create(artist='Apple', title='Pie')
+        test_song = {
+            'artist': 'Apple',
+            'title': 'Pie'
+        }
+        add_song(**test_song)
```

Comment:
Deze setUp gebruik je volgens mij nog maar in een test van deze klasse.  Dan is het logischer de bijbehorende setup in die test te doen i.p.v. voor de hele klasse

Reply:
Good find, I changed it accordingly